lib/rhc/commands/app.rb

```diff
@@ -376,9 +376,20 @@ def tidy(app)
     syntax "<app> [--namespace NAME]"
     takes_application :argument => true
     def enable_ha(app)
-      app_action :enable_ha
+      rest_app = find_app
+
+      scaleup_needed = false
+      gear_groups = rest_app.gear_groups
+      gear_groups.each do |gg|
+        scaleup_needed = true if gg.attributes["scales_from"] > 1
```

Comment:
If this is true (we have the ability to identify a pain point), why not just prompt the user and ask them if they would like to scale up (+1) as well?

Or give them an option to scale down (to 1 gear) and continue (enable ha)?

With this code in place to see if you should tell a user what to do, the option exsists to allow the command to scale up or down either before or after the enablement (accordingly), and not make the users do these operations as extra steps.